theme/404.html

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
 <div class="page-not-found">
-    <img src="/images/404.png" alt="Not found" />
+    <img src="{{ base_url }}/images/404.png" alt="Not found" />
```

Comment:
Fix double slashes (`src="/en/latest//images/404.png"` → `src="/en/latest/images/404.png"`)
```suggestion
    <img src="{{ base_url }}{% if base_url|last != "/" %}/{% endif %}images/404.png" alt="Not found" />
```

Reply:
Contrary to search.html, `{{ base_url }}` is not relative but absolute in 404.html to have the same HTML whatever the wrong URL is.